Localization/de-DE/home.yaml

```
@@ -41,6 +53,49 @@ Für technische Hilfe: ",
 #URLs - Facebook and Twitter have localised name in certain languages:
 "foldingforum.org",
 "Facebook",
-"Twitter"
+"Twitter",
+
+#LATEST POSTS bold text
+#"LATEST POSTS",
+"NEUESTE ARTIKEL",
+
+# WHY WE NEED YOU bold text
+#"WHY WE NEED YOU",
+"WARUM WIR SIE BRAUCHEN",
+
+# Associated text
+#"Folding@home is a project focused on disease research. The problems we're solving require so many computer calculations and we need your help to find the cures!",
+"Folding@home ist ein Projekt, das sich auf die Erforschung von Krankheiten konzentriert. Die Problemlösungen erfordern eine so große Menge an Berechnungen, dass wir Ihre Hilfe brauchen, um Heilmittel zu finden!",
+
+#FIGHT DESIEASES TOGETHER WITH US bold text
+#"FIGHT DISEASES TOGETHER WITH US.",
+"BEKÄMPFT KRANKHEITEN GEMEINSAM MIT UNS",
+
+#Associated text
+#"The Folding@home software allows you to share your unused computer power with us so that we can research even more potential cures.",
+"Die Folding@home Software erlaubt es Ihnen, uns Ihre ungenutzte Rechenleistung zur Verfügung zu stellen, so dass wir noch mehr mögliche Heilmittel untersuchen können.",
+
+#Associated text in pentagon
+#"1 in a million",
+"1 unter Millionen",
+
+#FOLDING? bold text
+"FALTEN?",
+
+#Associated text
+#"Folding refers to the way human protein folds in the cells that make up your body. We rely on the proteins to keep us healthy and they assemble themselves by folding. But when they misfold, there can be serious consequences to a person's health.",
+"Falten bezieht sich auf die Weise, wie sich menschliche Proteine in den Zellen, aus denen Ihr Körper besteht, verformen. Unsere Gesundheit ist auf Proteine angewiesen, und sie fügen sich durch das Falten zusammen. Aber wenn sie sich falsch falten kann dies ersthafte Folgen für die Gesundheit einer Person haben.",
+
+#Associated square button
+#"LEARN MORE",
+"ERFAHRE MEHR",
+
+#TEAM bold text
+#TEAM,
+"TEAM",
+
+#Associated text
+#"Folding@home is now based at the Washington University in St. Louis School of Medicine, under the directorship of  Dr. Greg Bowman. Drs. John Chodera (MSKCC) and Vince Voelz (Temple University) are also active in helping manage the project. Together, their three labs are the primary drivers of Folding@home."
+"Folding@home ist derzeit bei der Washington University in St. Louis School of Medicine zu Hause und steht unter der Leitung von Dr. Greg Bowman. Die Dres. John Chodera (MSKCC) und Vince Voelz (Temple University) unterstützen das Projektmanagement ebenfalls aktiv. Zusammen sind ihre drei Labore die hauptsächlichen Treiber von Folding@home."
```

"Zusammen sind ihre drei Labore die hauptsächlichen Treiber von Folding@home."
sounds pretty weird. i cant think of a better option right now though.

---

Zusammen sind ihre drei Laboratorien die Haupttreiber von Folding@home.

---

```suggestion
"Folding@home ist derzeit bei der Washington University in St. Louis School of Medicine zu Hause und steht unter der Leitung von Dr. Greg Bowman. Die Dres. John Chodera (MSKCC) und Vince Voelz (Temple University) unterstützen das Projektmanagement. Ihre drei Labore treiben Folding@home hauptsächlich voran."
```
I think we can remove that _ebenfalls aktiv_, because _unterstützen_ is already active in german

---

@max-dau, agreed, it sounds robotic... is it the "Treiber"? Maybe "maßgebliche Unterstützer"?

@dsdel, I kept that in to show that their support is "hands-on", not "we watch and like what we see" 😃 

I like @michael-raps' suggestion best - any objections?

---

If they support, it is active - there is no passiv support without declaration.
Haupttreiber sounds wired to me : )

---

i prefer dsdel's. Anything-Treiber sounds unnatural to me.